flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/catalog/GravitinoCatalogManager.java

```diff
@@ -54,8 +61,35 @@ private GravitinoCatalogManager(
     this.gravitinoUri = gravitinoUri;
     this.metalakeName = metalakeName;
     this.gravitinoClientConfig = gravitinoClientConfig;
-    this.gravitinoClient =
-        GravitinoAdminClient.builder(gravitinoUri).withClientConfig(gravitinoClientConfig).build();
+
+    String authType = gravitinoClientConfig.get(GravitinoCatalogStoreFactoryOptions.AUTH_TYPE);
+
+    // Only OAuth is explicitly configured; otherwise follow Flink security (Kerberos if enabled,
```

Comment:
could you provide more context about this?

Reply:
Gravitino client authentication type should follow the Flink first. 
If Flink used simple mode, Gravitino used simple mode. If Flink used Kerberos mode, Gravitino used Kerberos mode. 
Flink didn't support OAuth mode, so we should allow to set the oauth explicitly.

Reply:
got it